src/mudlet.cpp

```diff
@@ -500,8 +500,45 @@ mudlet::mudlet()
     initEdbee();
 }
 
+QPointer<QSettings> mudlet::getQConfig()
+{
+    QPointer<QSettings> config;
+    // Try for a mudlet config file, first, and check whether a custom
+    // path for the configuration is given
+    config = new QSettings(QCoreApplication::applicationDirPath() + "/config.ini", QSettings::IniFormat);
+    // if the settings file 'config.ini' (relative to the mudlet
+    // executable file) does not exist (read: it doesn't contain the
+    // setting "configPath"), try to load the config from the default
+    // paths
+    if (!config->contains("configPath")) {
+        config = new QSettings(QSettings::IniFormat, QSettings::UserScope, "mudlet", "config");
```

Comment:
:earth_africa: **I18n**: Please write non-GUI translatable strings inside a `QLatin1String(`...`)` wrapper if there is one for the method concerned or, as there does not seem to be ones for the `QSettings` class, you should use the other one, `QStringLiteral(`...`)` so that we do not have to decide later on whether it needs to be fed to the translation effort now being initiated. :innocent:

Reply:
Oh, I see! I'll do that.

What I have in mind is to store the configuration directory setting (`configPath`) in its own separate `QSettings` file so that Mudlet's configuration all stays in one directory. Because `configPath` could refer to another directory (when the option `Select a directory` is chosen by the user), if it were in `Mudlet.conf`, that would mean separating `Mudlet.conf` from the configuration directory.